### ISSUE TITLE
helpers/php_container throws deprecation notices for PHP 8.1

### ISSUE THREAD

**Issue by codyfinegan** [totara/totara-docker-dev#195]
PHP 8.1 has deprecated passing null into built-in functions that accept strings (such as explode, trim etc...). A deprecated message will be shown when this happens.

```
PHP Deprecated:  trim(): Passing null to parameter #1 ($string) of type string is deprecated in /home/cody/Projects/Docker/cody-docker/bin/helpers/php_container.php on line 17
PHP Deprecated:  trim(): Passing null to parameter #1 ($string) of type string is deprecated in /home/cody/Projects/Docker/cody-docker/bin/helpers/php_container.php on line 19
Creating network "totara" with the default driver
Creating network "cody-docker_default" with the default driver
Creating totara_php80_debug ... done
Creating totara_php80       ... done
Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated in /home/cody/Projects/Docker/cody-docker/bin/helpers/php_container.php on line 17 Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated in /home/cody/Projects/Docker/cody-docker/bin/helpers/php_container.php on line 19 php-8.0 e0c08e11f191 8.0
/home/cody/Projects/Docker/cody-docker/bin/tdb: eval: line 104: syntax error near unexpected token `('
/home/cody/Projects/Docker/cody-docker/bin/tdb: eval: line 104: `docker exec trim(): /bin/zsh -i -c 'cd /var/www/totara/src//testing && config_var dataroot dbtype dbhost dbname dbuser dbpass''
Database config variables are missing from config.php
```

**To Reproduce**
- `php -v` should show 8.1
- Make sure no totara_php container running (run `tdown` to make sure)
- Setup your config.php normally with database credentials
- Run `tdb create`

**Expected behavior**
The PHP container is booted up & the database is created (it should auto-discover the specific PHP container).

**Desktop (please complete the following information):**
 - OS: Windows (WSL2)
 - PHP 8.1 CLI

**Additional context**
See https://www.php.net/manual/en/migration81.deprecated.php#migration81.deprecated.core.null-not-nullable-internal

**Comment by codyfinegan** [comment]
Actual fix is just where we call trim/ or any string functions, check that the input is a string & not a null.

**Comment by exception0x876** [comment]
@codyfinegan please see https://github.com/totara/totara-docker-dev/pull/200

**Comment by derschatta** [comment]
fixed by #200
